ddpui/api/dbt_automation_api.py

```
@@ -0,0 +1,235 @@
+    ]
+
+    # Return the DAG as a dictionary
+    return {
+        "nodes": node_list,
+        "edges": edge_list,
+    }
+
+
+@dbtautomation_router.delete("/node/{node_uuid}/", auth=auth.CustomAuthMiddleware())
+@has_permission(["can_create_dbt_model"])
+def post_chain_operation_node(request, node_uuid: str, canvas_lock_id: str = None):
```

Ruff (0.8.2)

Redefinition of unused `post_chain_operation_node` from line 87

(F811)